lib/disk.js

```diff
@@ -57,10 +57,12 @@ module.exports.readFilesystem = function(archive) {
 };
 
 module.exports.readFile = function(filesystem, info) {
-  var fd = fs.openSync(filesystem.src, 'r');
   var buffer = new Buffer(info.size);
-  fs.readSync(fd, buffer, 0, info.size,
-              8 + filesystem.headerSize + parseInt(info.offset));
-  fs.closeSync(fd);
+  if(info.size > 0) {
```

Comment:
Needs a space here: `if(` => `if (`.

And can you add some comment of the purpose in the code?